### PR TITLE
Create issue templates 

### PR DESCRIPTION
**Pull request by ravjotbrar**
### Summary

Created issue templates for bug reports and enhancement requests. In this same PR, I also moved three individual tests to the basic tests folder, so that these are easier to run (until we fix https://github.com/dremio/dbt-dremio/issues/91) 

### Description

Bug reports now outline information from the reporter's perspective:
1. Current Behaviour
2. Expected Behaviour
3. Steps to reproduce
4. Adapter Version
5. Relevant logs

Enhancement requests are simpler with:
1. Description of the enhancement
2. Justification for the enhancement

### Test Results

N/A

### Changelog

N/A

### Related Issue

https://github.com/dremio/dbt-dremio/issues/128

### CODE REVIEW

**Comment by ArgusLi** [comment]
I like that we're reorganising tests, but I think we should match the structure of dbt-core.

**Comment by ravjotbrar** [comment]
> I like that we're reorganising tests, but I think we should match the structure of dbt-core.

I agree with that sentiment. At the moment it's just easier running the tests manually when they're in two separate folders. I suppose we could also write a script that runs them separately as well (only until we get the issue resolved where we can't run all the func tests at once).

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      placeholder: |
+        1. In this environment...
+        2. With these relevant config options...
+        3. Run '...'
+        4. See error...
+  - type: dropdown
+    id: version
+    attributes:
+      label: Adapter Version
+      description: What version of our software are you running?
```
Comment (ArgusLi) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]: Maybe "what version of dbt-dremio"?

Comment (ArgusLi) [reply .github/ISSUE_TEMPLATE/bug_report.yml:39]: Also should we have one for dremio version as well?

**Review by ArgusLi — COMMENTED** [review]
LGTM apart from that - just some nits

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (ravjotbrar) [reply .github/ISSUE_TEMPLATE/bug_report.yml:39]: Good suggestions. Will add them in.

**Review by jlarue26 — APPROVED** [review]
Looks great, thanks!

**Pull request merged**